test_reproducibility.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+import tempfile
+
+import pandas as pd
+from misc.utils import set_seed
+from runners import main_runner
+
+
+TCASE_CONFIG = {
+    "model": {"parameters":  {"protocols": ["OR", "AND"], "p_values": [0.1, 0.35, 0.9]}},
+    "networks": ["toy_network"],
+    "run": {"repetitions": 3, "random_seed": 43, "average_results": False},
+    "logging": {"compress_to_zip": False, "out_dir": None}
+}
+TCASE_CSV_NAMES = [
+    Path("proto-AND--p-0.1--net-toy_network.csv"),
+    Path("proto-AND--p-0.9--net-toy_network.csv"),
+    Path("proto-AND--p-0.35--net-toy_network.csv"),
+    Path("proto-OR--p-0.1--net-toy_network.csv"),
+    Path("proto-OR--p-0.9--net-toy_network.csv"),
+    Path("proto-OR--p-0.9--net-toy_network.csv"),
+]
+
+
+def compare_results(gt_dir: Path, test_dir: Path, csv_names: list[str]) -> None:
+    for csv_name in csv_names:
```

Comment:
Use parametric tests with `pytest.mark.parametrize`

Reply:
It most certainly looks parametric. You defined multiple test files and you perform the same assertion on all of them

Reply:
Well, it can be parametrised, but in fact I'm handling here only one experiment. And by experiment I understand generating the dataset according to provided configuration. There is one configuration, so one test case. You've been presumably confused by several output files that are checked. In fact, they're an output of a single run